### PR TITLE
Add astro-gala to conda-forge

### PR DESCRIPTION
**Pull request by mwcraig**
There is already a package called gala on conda-forge and PyPI, so this one is called `astro-gala`.

### CODE REVIEW

**Comment by mwcraig** [comment]
ping @bsipocz -- once this is merged we can update astropy/conda-channel-astropy#158 to remove everything except the package name from `requirements.yml`.

File: recipes/astro-gala/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
```
Comment (ocefpaf) [inline recipes/astro-gala/meta.yaml]: Any special reason to add the `--offline`?

File: recipes/astro-gala/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```
Comment (ocefpaf) [inline recipes/astro-gala/meta.yaml]: I am assuming it is not using numpy ABI (even though it is listed in the build section.)

File: recipes/astro-gala/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+{% set name = "astro-gala" %}
+{% set version = "0.2.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "83b319d1e05dbe0453671a3d28cadacb7f57bcf17e3f7e24441410b4cd1ff134" %}
+
+package:
+  name: '{{ name|lower }}'
```
Comment (ocefpaf) [inline recipes/astro-gala/meta.yaml]: Do we need the extra quotes?

File: recipes/astro-gala/meta.yaml (hunk 1 above)
Comment (bsipocz) [inline recipes/astro-gala/meta.yaml]: ``--offline`` was needed over there in https://github.com/astropy/conda-channel-astropy for all packages that relied on the astropy_helpers submodule.

File: recipes/astro-gala/meta.yaml (hunk 2 above)
Comment (mwcraig) [inline recipes/astro-gala/meta.yaml]: correct

**Comment by mwcraig** [comment]
The reason for `--offline` is that the astropy affiliated packages use a submodule called `astropy_helpers` during build, and that packages tries to download any patch updates unless `--offline`` is used. That attempted download causes an exception during conda build, which blocks downloads.

**Comment by ocefpaf** [comment]
Thanks for the explanation @mwcraig!

**Pull request merged**